pixi.toml

```
@@ -192,6 +192,7 @@ tomli-w = "*"
 xarray = "*"
 xmipy = "*"
 xugrid = "*"
+teamcity-messages = ">=1.32,<2"
```

```suggestion
teamcity-messages = "*"
```

Otherwise we currently have no way to get breaking updates. And with the readable diff like https://github.com/Deltares/Ribasim/pull/1873 we can easily see and double check major version updates.

---

I didn't know about that fancy diff, very cool!